src/core/server/systems/admin.ts

```diff
@@ -19,9 +19,13 @@ export class AdminController {
         }
 
         player.kick(`[Banned] ${reason}`);
-        Database.updatePartialData(player.accountData._id, { banned: true, reason }, Collections.Accounts);
-        Logger.info(`(${player.discord.id}) Has been banned from the server.`);
-        return true;
+        const result = Database.updatePartialData(player.accountData._id, { banned: true, reason }, Collections.Accounts);
+        if(result) {
```

Comment:
This if statement will always be `true` because `updatePartialData` is a promise.